### ISSUE TITLE
Editor not showing after upgrading to 1.2.5 on macOS

### ISSUE THREAD

**Issue by markron** [zsviczian/obsidian-excalidraw-plugin#91]
**Describe the bug**
First of all, thanks for the exceptional plugin.
After upgrading to version 1.2.5, opening a drawing (either in old or new format) does not open the Excalidraw editor anymore, the page remains blank. On the JavaScript console an error with message `TypeError: r.replaceAll is not a function` is thrown.
I encounter the problem only on macOS, on iOS and Android the plugin works just fine (I didn't try on other operating systems).

**To Reproduce**
1. Install the latest version of the plugin on a newly created vault on Obsidian for macOS;
2. Create and open a new drawing;
3. The page remains blank.

**Expected behavior**
The Excalidraw editor being shown when opening a drawing.

**Screenshots**
<img width="2032" alt="Screenshot 2021-07-12 at 12 01 19" src="https://user-images.githubusercontent.com/2474724/125269920-74e04e00-e309-11eb-8c4a-156bf0c248a2.png">

**Desktop**
 - OS: macOS Catalina 10.15.7
 - Obsidian Version: 0.12.10
 - Excalidraw Plugin Version: 1.2.5

**Comment by zsviczian** [comment]
Not good :(
Could this be a template issue? Do you have a template set up on your Mac?
Do you have a template set? What if you rename the template, or change the template setting to something different (e.g. a non-existent file)?

**Comment by markron** [comment]
Reinstalling Obsidian (deleting the app bundle from the Applications folder and downloading the latest version from the Obsidian website) solved the issue. I recently upgraded to the latest version of Obsidian using the built-in updates feature (Settings > About). I don't know how that update process works, but the error seemed to suggest an outdated Electron version (`String.prototype.replaceAll` is available from Chrome 85) or something like that. My installation was actually pretty old, I installed the application back when Obsidian was first announced. So in the end I don't think it had anything to do with templates (which I didn't configure at all) or your plugin specifically. Thanks anyway for the quick response!
